src/types/CompareCertReq.ts

```diff
@@ -28,5 +29,11 @@ export const deserializeCompareCertReq = (stream: VectorBufferStream): CompareCe
     throw new Error(`Unsupported CompareCertReqSerializable version ${version}`)
   }
   const obj: CompareCertReqSerializable = Utils.safeJsonParse(stream.readString())
+
+  const errors = verifyPayload('CompareCertReq', obj)
```

Comment:
Why isnt `AJVSchemaEnum...` being used here like it is in ApoptosisProposalReq.ts?

Reply:
Fixed.